servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpSetCookie.java

```diff
@@ -45,13 +50,14 @@
      * if/else block we lean on {@link HttpHeaders} which provides an associative array that compares keys in a case
      * case-insensitive manner.
      */
-    private static final HttpHeaders AV_FIELD_NAMES = new DefaultHttpHeaders(8, false, false);
+    private static final HttpHeaders AV_FIELD_NAMES = new DefaultHttpHeaders(16, false, false);
```

Comment:
I just wondered why 16 now... Seems like 8 should be enough ?

Reply:
8 and 16 are both functional, but 8 lead to overlap in a bucket between 2 entries and 16 makes it so there is no overlap.

Reply:
javadoc mentions `8` which is stale now.

Reply:
javadoc fixed